keanu-python/build.gradle

```diff
@@ -111,7 +111,7 @@ task populateExclusionPaths {
 }
 
 task formatApply {
-    dependsOn preparePythonEnvironment
+    dependsOn installPipenv
```

Comment:
Does it not also depend on `installDependencies`? Otherwise `yapf` won't be available to the pipenv.

Reply:
Yes it does - that's frustrating because `installDependencies` is slow.

I've had a look at the various `pipenv` commands but it doesn't appear possible make it run really fast (e.g. tell it to only upgrade dependencies if their versions have changed)